lib/sidekiq/client.rb

```diff
@@ -193,7 +193,7 @@ def atomic_push(conn, payloads)
         q = payloads.first['queue']
         to_push = payloads.map { |entry| Sidekiq.dump_json(entry) }
         conn.sadd('queues'.freeze, q)
-        conn.lpush("queue:#{q}", to_push)
+        conn.lpush("queue:#{q}".freeze, to_push)
```

Comment:
Yes please do not do that. We dynamically create queues.

Sent from my mobile device
On Apr 5, 2015 11:55 AM, "Mike Perham" <notifications@github.com> wrote:

> In lib/sidekiq/client.rb
> <https://github.com/mperham/sidekiq/pull/2288#discussion_r27778277>:
>
> > @@ -193,7 +193,7 @@ def atomic_push(conn, payloads)
> >          q = payloads.first['queue']
> >          to_push = payloads.map { |entry| Sidekiq.dump_json(entry) }
> >          conn.sadd('queues'.freeze, q)
> > -        conn.lpush("queue:#{q}", to_push)
> > +        conn.lpush("queue:#{q}".freeze, to_push)
>
> My policy is don't freeze dynamically created strings. What do you think?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/mperham/sidekiq/pull/2288/files#r27778277>.
>